# Question: How many people didn't survive on the Titanic?

## This would be a census study because we have access to all the passengers on board the Titanic and we can see what class they're in and who didn't survive and who did. We also have the entitre population in this dataset.

In [13]:
ser = pd.Series(range(5), index=list("abcde"))

ser.loc[["a", "c", "e"]]

a    0
c    2
e    4
dtype: int64

In [46]:
Titanic["Survived"]

survived = (Titanic[Titanic["Survived"] == 1])

In [32]:
import pandas as pd

df = pd.read_csv('titanic.csv')

print(df.groupby(['Survived', 'Pclass']).size())

Survived  Pclass
0         1          80
          2          97
          3         372
1         1         136
          2          87
          3         119
dtype: int64


In [47]:
count = len(survived)
print("Survived", count)

Survived 342


# Conclusion
There are a total of 342 who survived the Titanic and people 1,500 people died on the Titanic out of the 2.224 people who were on the Titanic total and that was over half of the entire ship.

In [9]:
import pandas as pd

Titanic = pd.read_csv('titanic.csv')

Titanic.head()

,PassengerId,Survived,Pclass,Name,Sex,Age,SibSp,Parch,Ticket,Fare,Cabin,Embarked
0,1,0,3,"Braund, Mr. Owen Harris",male,22.0,1,0,A/5 21171,7.2500,NaN,S
1,2,1,1,"Cumings, Mrs. John Bradley (Florence Briggs Th...",female,38.0,1,0,PC 17599,71.2833,C85,C
2,3,1,3,"Heikkinen, Miss. Laina",female,26.0,0,0,STON/O2. 3101282,7.9250,NaN,S
3,4,1,1,"Futrelle, Mrs. Jacques Heath (Lily May Peel)",female,35.0,1,0,113803,53.1000,C123,S
4,5,0,3,"Allen, Mr. William Henry",male,35.0,0,0,373450,8.0500,NaN,S


In [10]:
Titanic.info()

<class 'pandas.DataFrame'>
RangeIndex: 891 entries, 0 to 890
Data columns (total 12 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   PassengerId  891 non-null    int64  
 1   Survived     891 non-null    int64  
 2   Pclass       891 non-null    int64  
 3   Name         891 non-null    str    
 4   Sex          891 non-null    str    
 5   Age          714 non-null    float64
 6   SibSp        891 non-null    int64  
 7   Parch        891 non-null    int64  
 8   Ticket       891 non-null    str    
 9   Fare         891 non-null    float64
 10  Cabin        204 non-null    str    
 11  Embarked     889 non-null    str    
dtypes: float64(2), int64(5), str(5)
memory usage: 118.9 KB
